### PR TITLE
Spike/813 vulnerability assessment with trivy

### PR DESCRIPTION
**Pull request by trottomv**

### CODE REVIEW

**Review by niccolomineo — DISMISSED** [review]

**Review by pauloxnet — CHANGES_REQUESTED** [review]
Copy here changes from https://github.com/20tab/django-continuous-delivery/pull/257 (see comments)

**Comment by trottomv** [comment]
> Copy here changes from #257 (see comments)

ok

File: {{cookiecutter.project_dirname}}/.gitlab-ci.yml [hunk 1]
```diff
@@ -326,6 +330,43 @@ build_production:
...
+    - >
+      docker run --rm
+      --entrypoint=""
+      --env CUSTOM_GID=`id -g`
+      --env CUSTOM_UID=`id -u`
+      --env GIT_STRATEGY=none
+      --env REPORT_NAME=report
+      --env TARGET_IMAGE=${CI_REGISTRY_IMAGE}/${CI_PROJECT_PATH}:${VERSION_REF}
+      --env TRIVY_AUTH_URL="${CI_REGISTRY}"
+      --env TRIVY_CACHE_DIR=.trivicache/
```
Comment (pauloxnet) [inline {{cookiecutter.project_dirname}}/.gitlab-ci.yml:350]:
```suggestion
      --env TRIVY_CACHE_DIR=.trivycache/
```

**Review by pauloxnet — CHANGES_REQUESTED** [review]

File: {{cookiecutter.project_dirname}}/.gitlab-ci.yml [hunk 2]
```diff
@@ -326,6 +330,42 @@ build_production:
...
+      --env TRIVY_AUTH_URL="${CI_REGISTRY}"
+      --env TRIVY_CACHE_DIR=.trivycache/
+      --env TRIVY_NO_PROGRESS=true
+      --env TRIVY_PASSWORD="${CI_REGISTRY_PASSWORD}"
+      --env TRIVY_USERNAME="${CI_REGISTRY_USER}"
+      --name trivy-{{ cookiecutter.project_slug }}-{{ cookiecutter.service_slug }}-${CI_JOB_ID}
+      --volume ${PWD}/scripts/trivy.sh:/app/trivy.sh
+      aquasec/trivy /app/trivy.sh
+  after_script:
+    - docker cp trivy-{{ cookiecutter.project_slug }}-{{ cookiecutter.service_slug }}-${CI_JOB_ID}:/tmp/${REPORT_NAME}.html trivy/report.html || true
```
Comment (trottomv) [inline {{cookiecutter.project_dirname}}/.gitlab-ci.yml:355]:
```suggestion
    - docker cp trivy-{{ cookiecutter.project_slug }}-{{ cookiecutter.service_slug }}-${CI_JOB_ID}:/tmp/report.html trivy/report.html || true
```

**Comment by trottomv** [comment]
consider it a spike for now, postponed

**Pull request closed without merging**